### DISCUSSION TITLE
No Audio after convert from .ts to .mp4

### DISCUSSION THREAD

**Discussion by erkan-burkan** [subhra74/xdm#1001] [Q&A]
Xdm downloaded a video file from website as .ts format,  though the source file is playing well in the website through the browser with video and audio, The downloaded file has Video with no Audio.The properties of the downloaded file has no audio properties or setting and it is all dim.

Any fast solution.

Many thanks in advance

**Comment by Budgie2** [comment]
I  have the same question.  Are there answers out there?

**Reply by Budgie2** [reply]
This is becoming serious as I cannot work without audio.  I have upgraded to the beta app and may not have new setup quite correct but my downloads show as .ts files but do not include any video.  Something is wrong but I am unable to diagnose without some help.

**Reply by linpoui** [reply]
m4a
